package.json

```diff
@@ -210,6 +210,7 @@
     "eslint-config-fb-strict": "^20.0.3",
     "eslint-config-fbjs": "^1.1.1",
     "eslint-plugin-eslint-comments": "^2.0.1",
+    "eslint-plugin-custom": "file:eslint-plugin",
```

Comment:
I'm not sure this will work with our internal Yarn offline cache that well, unfortunately.

Reply:
Looks like nuclide uses this to register custom plugins, that would probably work instead or using a file dependency. https://github.com/facebook/nuclide/blob/693623fd6e2124b094ab4d4d5fad15a67496c6b7/.eslintrc.js#L513-L514